test/e2e/configurable_route_test.go

```diff
@@ -369,6 +373,7 @@ func TestConfigurableRouteNoConsumingUserNoRBAC(t *testing.T) {
 				RelatedObjects:   []configv1.ObjectReference{{}},
 			},
 		},
+		DefaultPlacement: configv1.DefaultPlacementControlPlane,
 	}
 	if err := eventuallyUpdateIngressStatus(t, ingress.Status); err != nil {
 		t.Errorf("error updating ingress.status: %v", err)
```

Comment:
You shouldn't need to specify `DefaultPlacement` now that https://github.com/openshift/cluster-config-operator/pull/243 has merged.

Reply:
Cool, removed the last commit.